source/octf/cli/Executor.cpp

```diff
@@ -492,5 +475,32 @@ void Executor::setupOutputsForCommandsLogs() const {
     log::cout << log::enable << log::json << log::prefix << prefix;
 }
 
+void Executor::checkIsKeyRepeated(const std::string &shortKey,
+                                  const std::string &longKey) const {
+    if (m_localCmdSet->hasCmd(longKey)) {
+        throw Exception("Cannot add command because of long key conflict: " +
+                        longKey);
+    }
+
+    if (!shortKey.empty() && m_localCmdSet->hasCmd(shortKey)) {
+        throw Exception("Cannot add command because of short key conflict: " +
+                        shortKey);
+    }
+
+    for (const auto iter : m_modules) {
+        if (iter.first.getLongKey() == longKey) {
```

Comment:
as above

Reply:
In auto-CLI empty long key is not allowed